hypothesis-python/src/hypothesis/internal/observability.py

```diff
@@ -31,11 +33,16 @@ def deliver_json_blob(value: dict) -> None:
         callback(value)
 
 
+class ObservabilityPredicate(TypedDict):
+    satisfied: int
+    unsatisfied: int
```

Comment:
Name seems to suggest that this is a callable; maybe `ObservabilityPredicateCounts`, or something similar but shorter?